--- /usr/libexec/backup-cron/backup-cron_functions.sh
+++ /usr/libexec/backup-cron/backup-cron_functions.sh
@@ -222,7 +222,7 @@
 
   for domain in ${DOMAINS}; do
     # Búsqueda de imágenes de discos utilizados por cada dominio (maquina virtual).
-    IMAGES=$(/usr/bin/virsh domblklist ${domain} | awk -F \  /^vd*/'{print $2}')
+    IMAGES=$(/usr/bin/virsh domblklist ${domain} | awk -F \  /^[sv]d*/'{print $2}')
     message_syslog "${NAME}" "Comenzando el respaldo para el dominio ${domain}."
 
     # Creación de instantáneas para los discos del dominio.
